### PR TITLE
FEC-12555 Player sample iOS updated deployment target to 12.0

### PR DESCRIPTION
**Pull request by chausov**
FEC-12555
Player sample iOS updated deployment target to 12.0

### CODE REVIEW

**Comment by x-NR-x** [comment]
What about the BasicSample?
Is this only for the iOS samples or also needed for the tvOS samples?

**Comment by chausov** [comment]
> What about the BasicSample?
> Is this only for the iOS samples or also needed for the tvOS samples?

In Basic we do not have cast implementation so it is not required to set it to 12.0

**Comment by x-NR-x** [comment]
> > What about the BasicSample?
> > Is this only for the iOS samples or also needed for the tvOS samples?
> 
> In Basic we do not have cast implementation so it is not required to set it to 12.0 Same with tvOS.

In that case, because it was done only for the cast, that is ok.
Next time please mention it here and/or the ticket.

File: OTTSample/OTTSample.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -481,15 +481,17 @@
 				CODE_SIGN_STYLE = Automatic;
 				DEVELOPMENT_TEAM = X8RDCFJK95;
 				INFOPLIST_FILE = OTTSample/Info.plist;
-				IPHONEOS_DEPLOYMENT_TARGET = 11.0;
+				IPHONEOS_DEPLOYMENT_TARGET = 12.0;
 				LD_RUNPATH_SEARCH_PATHS = (
 					"$(inherited)",
 					"@executable_path/Frameworks",
 				);
+				MACOSX_DEPLOYMENT_TARGET = 12.0;
```
Comment (x-NR-x) [inline OTTSample/OTTSample.xcodeproj/project.pbxproj:489]: Why was this added?
The sample is only for iOS.
Please remove.

File: OTTSample/OTTSample.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -481,15 +481,17 @@
...
 				PRODUCT_BUNDLE_IDENTIFIER = com.kaltura.KPOTTSample;
 				PRODUCT_NAME = "$(TARGET_NAME)";
 				SWIFT_VERSION = 5.0;
 				TARGETED_DEVICE_FAMILY = "1,2";
+				TVOS_DEPLOYMENT_TARGET = 12.0;
```
Comment (x-NR-x) [inline OTTSample/OTTSample.xcodeproj/project.pbxproj:494]: Why was this added?
The sample is only for iOS.
Please remove.

File: OTTSample/OTTSample.xcodeproj/project.pbxproj [hunk 3]
```diff
@@ -502,15 +504,17 @@
 				CODE_SIGN_STYLE = Automatic;
 				DEVELOPMENT_TEAM = X8RDCFJK95;
 				INFOPLIST_FILE = OTTSample/Info.plist;
-				IPHONEOS_DEPLOYMENT_TARGET = 11.0;
+				IPHONEOS_DEPLOYMENT_TARGET = 12.0;
 				LD_RUNPATH_SEARCH_PATHS = (
 					"$(inherited)",
 					"@executable_path/Frameworks",
 				);
+				MACOSX_DEPLOYMENT_TARGET = 12.0;
```
Comment (x-NR-x) [inline OTTSample/OTTSample.xcodeproj/project.pbxproj:512]: Why was this added?
The sample is only for iOS.
Please remove.

File: OTTSample/OTTSample.xcodeproj/project.pbxproj [hunk 4]
```diff
@@ -502,15 +504,17 @@
...
 				PRODUCT_BUNDLE_IDENTIFIER = com.kaltura.KPOTTSample;
 				PRODUCT_NAME = "$(TARGET_NAME)";
 				SWIFT_VERSION = 5.0;
 				TARGETED_DEVICE_FAMILY = "1,2";
+				TVOS_DEPLOYMENT_TARGET = 12.0;
```
Comment (x-NR-x) [inline OTTSample/OTTSample.xcodeproj/project.pbxproj:517]: Why was this added?
The sample is only for iOS.
Please remove.

File: OVPSample/OVPSample.xcodeproj/project.pbxproj [hunk 5]
```diff
@@ -481,15 +481,17 @@
 				CODE_SIGN_STYLE = Automatic;
 				DEVELOPMENT_TEAM = X8RDCFJK95;
 				INFOPLIST_FILE = OVPSample/Info.plist;
-				IPHONEOS_DEPLOYMENT_TARGET = 11.0;
+				IPHONEOS_DEPLOYMENT_TARGET = 12.0;
 				LD_RUNPATH_SEARCH_PATHS = (
 					"$(inherited)",
 					"@executable_path/Frameworks",
 				);
+				MACOSX_DEPLOYMENT_TARGET = 12.0;
```
Comment (x-NR-x) [inline OVPSample/OVPSample.xcodeproj/project.pbxproj:489]: Why was this added?
The sample is only for iOS.
Please remove.

File: OVPSample/OVPSample.xcodeproj/project.pbxproj [hunk 6]
```diff
@@ -481,15 +481,17 @@
...
 				PRODUCT_BUNDLE_IDENTIFIER = com.kaltura.KPOVPSample;
 				PRODUCT_NAME = "$(TARGET_NAME)";
 				SWIFT_VERSION = 5.0;
 				TARGETED_DEVICE_FAMILY = "1,2";
+				TVOS_DEPLOYMENT_TARGET = 12.0;
```
Comment (x-NR-x) [inline OVPSample/OVPSample.xcodeproj/project.pbxproj:494]: Why was this added?
The sample is only for iOS.
Please remove.

File: OVPSample/OVPSample.xcodeproj/project.pbxproj [hunk 7]
```diff
@@ -502,15 +504,17 @@
 				CODE_SIGN_STYLE = Automatic;
 				DEVELOPMENT_TEAM = X8RDCFJK95;
 				INFOPLIST_FILE = OVPSample/Info.plist;
-				IPHONEOS_DEPLOYMENT_TARGET = 11.0;
+				IPHONEOS_DEPLOYMENT_TARGET = 12.0;
 				LD_RUNPATH_SEARCH_PATHS = (
 					"$(inherited)",
 					"@executable_path/Frameworks",
 				);
+				MACOSX_DEPLOYMENT_TARGET = 12.0;
```
Comment (x-NR-x) [inline OVPSample/OVPSample.xcodeproj/project.pbxproj:512]: Why was this added?
The sample is only for iOS.
Please remove.

File: OVPSample/OVPSample.xcodeproj/project.pbxproj [hunk 8]
```diff
@@ -502,15 +504,17 @@
...
 				PRODUCT_BUNDLE_IDENTIFIER = com.kaltura.KPOVPSample;
 				PRODUCT_NAME = "$(TARGET_NAME)";
 				SWIFT_VERSION = 5.0;
 				TARGETED_DEVICE_FAMILY = "1,2";
+				TVOS_DEPLOYMENT_TARGET = 12.0;
```
Comment (x-NR-x) [inline OVPSample/OVPSample.xcodeproj/project.pbxproj:517]: Why was this added?
The sample is only for iOS.
Please remove.

**Review by x-NR-x — CHANGES_REQUESTED** [review]

**Review by giladna — APPROVED** [review]

**Review by x-NR-x — APPROVED** [review]

**Pull request merged**